Go/Go.sublime-syntax

```diff
@@ -719,11 +719,20 @@ contexts:
 
   match-keyword-break:
     - match: \bbreak\b
-      scope: keyword.control.go
+      scope: keyword.control.flow.break.go
 
   match-keyword-case:
     - match: \bcase\b
-      scope: keyword.control.go
+      scope: keyword.control.switch.case.go
```

Comment:
```suggestion
      scope: keyword.control.conditional.case.go
```

According to other syntaxes in this repo `switch..case..default` scopes are organized within `keyword.control.conditional`.

Reply:
I just went through https://github.com/sublimehq/Packages/issues/1228 again and the state this (and other RFC) discussions are in is just a little said with no definitive conclusion available and the information spread throughout the discussion. I wish I had more time at hand to get back to SNG and properly compile the results of these discussions into an encyclopedia you can look up …